cluster/gce/trusty/node.yaml

```diff
@@ -162,8 +163,16 @@ start on stopped kube-env
 
 script
 	. /etc/kube-env
-	# If kubelet or kube-proxy is not installed in the image, pull release binaries and put them in /usr/bin.
-	if ! which kubelet > /dev/null || ! which kube-proxy > /dev/null; then
+	# If both kubelet and kube-proxy test args are set, this is a testing cluster.
+	# We pull binaries and put them in /usr/local/bin. If it is not a testing
```

Comment:
If /usr/bin may not be writable, shouldn't we be writing to /usr/local/bin whether or not it's a testing cluster? My question wasn't why you chose /usr/local/bin but rather why you chose to have it depend on two vars being set. Wouldn't it be easier to have it do the same thing in a test cluster and a non-test cluster?